arbitrum-docs/partials/_troubleshooting-building-partial.md

```diff
@@ -147,7 +147,18 @@ Once upon a time, Arbitrum developers were required to download supplemental pac
 
 
 
-### Is there any testnet or devnet available on Arbitrum? {#is-there-any-testnet-or-devnet-available-on-arbitrum}
+### What is a testnet or a devnet? {#what-is-a-testnet-or-a-devnet}
+<p>Testnets (or devnets) primarily serve developers who want to test out the applications they're building without having to use any real mainnet funds.</p>
+
+<p>Arbitrum testnets have the same full feature-set as the mainnet networks. They are also "true" L2s that run on top of public testnet Layer 1s, using their underlying networks for security and settlement.</p>
+
+<p>Users can bridge any asset from the underlying testnet L1 into an Arbitrum testnet (and back!), using the official <a href="https://bridge.arbitrum.io">bridge</a>.</p>
```

Comment:
`Users can bridge any asset from the Goerli testnet (L1) into the Nitro Goerli Rollup testnet (and back!), using the official bridge`